src/main/java/Duke.java

```diff
@@ -1,10 +1,43 @@
+import java.util.Scanner;
+
 public class Duke {
     public static void main(String[] args) {
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        System.out.println("Hello there General Kenobi! I'm\n" + logo);
+        System.out.println("How may I be at your pleasure today?");
+        Scanner sc = new Scanner(System.in);
+        Task[] storedTasks = new Task[100];
+        int index = 0;
+        while (true) {
+            String input = sc.nextLine();
+            String[] splitInput = input.split(" ");
+            if(input.equals("list")){
+                System.out.println("\tHere are the tasks in your list:");
+                for(int i=0;i<index;i++){
```

Comment:
might be an arrowhead code, maybe you can make the printing of tasks into a function instead :)

Reply:
Yes great idea thanks!